docs/docs/local-https.md

```diff
@@ -8,50 +8,148 @@ Gatsby provides an easy way to use a local HTTPS server during development, than
+  If you choose not to install `certutil`, or the automatic install is not successful, you may get the following errors/prompts:
 
-    Unable to automatically install SSL certificate - please follow the
-    prompts at http://localhost:52175 in Firefox to trust the root certificate
-    See https://github.com/davewasmer/devcert#how-it-works for more details
-    -- Press <Enter> once you finish the Firefox prompts --
+  - If you use Chrome on Linux:
 
-If you wish to support Firefox (or Chrome on Linux), visit `http://localhost:52175` in Firefox and follow the point-and-click wizard. Otherwise, you may press enter without following the prompts. **Reminder: you'll only need to do this once per machine.**
+    ```text
+    WARNING: It looks like you have Chrome installed, but you specified
+    'skipCertutilInstall: true'. Unfortunately, without installing
+    certutil, it's impossible get Chrome to trust devcert's certificates
+    The certificates will work, but Chrome will continue to warn you that
+    they are untrusted.
+    ```
 
-Now open the development server at `https://localhost:8000` and enjoy the HTTPS goodness ✨. Of course, you may change the port according to your setup.
+  - If you have Firefox installed, `devcert` will try to utilize Firefox itself to trust the certificate
+
+    ```text
+    devcert was unable to automatically configure Firefox. You'll need to
+    complete this process manually. Don't worry though - Firefox will walk
+    you through it.
+
+    When you're ready, hit any key to continue. Firefox will launch and
+    display a wizard to walk you through how to trust the devcert
+    certificate. When you are finished, come back here and we'll finish up.
+    (If Firefox doesn't start, go ahead and start it and navigate to
+    http://localhost:52175 in a new tab.)
+
+    If you are curious about why all this is necessary, check out
+    https://github.com/davewasmer/devcert#how-it-works
+    <Press any key to launch Firefox wizard>
+    ```
+
+    - You can press enter here, and it will launch Firefox for you.
+
+    - If you wish to have trust support on Firefox, tell the point-and-click wizard `this certificate can identify websites`, and click OK. Otherwise, you may hit cancel and close the browser, then key return to finish building. **Reminder: you'll only need to do this once per machine.**
+
+### After `devcert` setup process
+
+You can open the development server at [https://localhost:8000](https://localhost:8000) and enjoy the HTTPS goodness ✨. Of course, you may change the port according to your setup.
 
 Find out more about [how devcert works](https://github.com/davewasmer/devcert#how-it-works).
 
+### Management of certificates generated by devcert
+
+If you want to do some maintenance/cleanup of the certificates generated by `devcert`, please refer to [devcert-cli](https://github.com/davewasmer/devcert-cli/blob/master/README.md)
```

Comment:
```suggestion
If you want to do some maintenance/cleanup of the certificates generated by `devcert`, please refer to the [devcert-cli](https://github.com/davewasmer/devcert-cli/blob/master/README.md)
```

Reply:
Should I maybe add "documentation" here?  "The" feels a bit weird here.

So, 
`refer to the [devcert-cli documentation]`
or
`refer to the [devcert-cli project]`

Reply:
Since cli stands for command line interface "the devcert-cli" should be accurate.